docs/contributing/code-style/angular.md

```diff
@@ -158,3 +158,41 @@ Some appropriate operators are:
   consider carefully if this is the right operator for your use case. mergeMap will flatten
   observables but not care about the order. If ordering is important use `concatMap`. If you only
   care about the latest value use `switchMap`.
+
+## Import statements
+
+We have a couple of guidelines for import statements, which are enforced using the eslint rules
+[`no-restricted-imports`](https://eslint.org/docs/latest/rules/no-restricted-imports),
+[`import/order`](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md),
+and
+[`import/no-restricted-paths`](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-restricted-paths.md).
+
+These rules aim to:
+
+- Prevent relative imports across package boundaries.
+- Restrict packages from importing application specific code.
+- Enforce a convention for the order of import statements.
+
+### Packages
```

Comment:
I think this sub-heading is unnecessary, you could remove it and promote the sub-sub headings below. It's a fairly short section so it doesn't need this level of nesting.

Reply:
Thank you for the feedback! I have updated to the suggestion here https://github.com/bitwarden/contributing-docs/pull/556/commits/bf6405ce1899c2c03ba9d94d92eecc27776c15a1